addons/binding/org.openhab.binding.squeezebox/README.md

```diff
@@ -87,10 +87,26 @@ rule "Garage Door Open Notification"
 when
     Item GarageDoorOpenNotification received command ON
 then
+    // Play the notification on the default sink
+    say("The garage door is open!", "voicerss:enUS")
+    // Play the notification on a specific sink
     say("The garage door is open!", "voicerss:enUS", "squeezebox:squeezeboxplayer:5919BEA2-764B-4590-BC70-D74DCC15491B:20cfbf221510")
 end
 ```
 
+And, you can play sounds from the conf/sounds directory.
```

Comment:
There were a couple places where blank lines were needed.  One day I'll learn to push Jekyll-friendly commits.  :-)